selfdrive/car/gm/interface.py

```diff
@@ -207,19 +204,16 @@ def _update(self, c):
                                                          GearShifter.eco, GearShifter.manumatic],
                                        pcm_enable=self.CP.pcmCruise)
 
-    if ret.vEgo < self.CP.minEnableSpeed:
+    # Enabling at a standstill with brake is allowed
+    # TODO: verify 17 Volt can enable for the first time at a stop and allow for all GMs
+    if ret.vEgo < self.CP.minEnableSpeed and not (ret.standstill and ret.brake >= 20 and
```

Comment:
@adeebshihadeh how do you feel about adding a regen field to carState? That way we can properly differentiate between regen brake and brake pedal? Also can check against each in test_models